assets/js/modules/tagmanager/components/common/SettingsUseSnippetSwitch.js

```diff
@@ -34,12 +34,19 @@ export default function SettingsUseSnippetSwitch() {
 		select( MODULES_TAGMANAGER ).getUseSnippet()
 	);
 
-	const description = useSnippet
-		? __( 'Site Kit will add the code automatically.', 'google-site-kit' )
-		: __(
-				'Site Kit will not add the code to your site.',
-				'google-site-kit'
-		  );
+	const description = (
+		<p>
+			{ useSnippet
+				? __(
+						'Site Kit will add the code automatically.',
+						'google-site-kit'
+				  )
+				: __(
+						'Site Kit will not add the code to your site.',
+						'google-site-kit'
+				  ) }
+		</p>
+	);
 
 	return <UseSnippetSwitch description={ description } />;
```

Comment:
Not that there's anything wrong with the above change, but I'm wondering why not make the same change as in [assets/js/modules/analytics-4/components/common/SettingsUseSnippetSwitch.js](https://github.com/google/site-kit-wp/pull/5115/files#diff-44b7a10d2c41b0536f8423800377958ab5cdfd9b7f24d096a530c6baa4440a7f), for the sake of consistency? I.e.,

```js
return <UseSnippetSwitch description={ <p>{ description }</p> } />;
```

Reply:
Good catch, fixed.